style.css

```diff
@@ -110,3 +112,487 @@ ul {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 95px;
+  height: 40px;
+  background: #fff;
+  border: 1px solid #8993a4;
+}
+
+.icon4-holder {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 61px;
+  height: 40px;
+  background: #fff;
+  border: 1px solid #8993a4;
+}
+
+.link-button :hover,
+.link-button2 :hover,
+.sub-bot :hover {
+  color: blue;
+}
+
+.link-button {
+  margin-bottom: 5%;
+  display: flex;
+  align-items: center;
+  padding: 12px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  border: none;
+  order: 4;
+}
+
+.ltext1 {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.card {
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("img2.png");
+  background-repeat: no-repeat;
+  line-height: 75px;
+  background-size: 327px 386px;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+.card1 {
+  order: 6;
+}
+
+.card2 {
+  order: 7;
+}
+
+.card3 {
+  order: 7;
+}
+
+.card4 {
+  order: 7;
+}
+
+.card5 {
+  order: 7;
+}
+
+.card6 {
+  order: 7;
+}
+
+.cardt {
+  display: flex;
+  width: 295px;
+  height: 81px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  padding-top: 30%;
+  padding-left: 5%;
+}
+
+.cardp {
+  display: flex;
+  padding-left: 5%;
+  width: 295px;
+  height: 87px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.cardb {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  column-gap: 10px;
+  margin: 0 0 30px 10px;
+}
+
+.bot {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.button1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 40px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  background-size: 10px 20px;
+}
+
+.button2 {
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 95px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+a {
+  text-decoration: none;
+}
+
+.button3 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 61px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.link-button2 {
+  display: flex;
+  align-items: center;
+  margin-bottom: 10%;
+  width: 327px;
+  height: 48px;
+  background: #ff6b00;
+  border: none;
+}
+
+.About-myself {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  background: #1c1a19;
+}
+
```

Comment:
- [ ] Great job with your design :clap:. Kindly add the `background-image` in the about section as seen in your chosen Figma template 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205450325-e9a4e64c-0dc3-4d69-9208-e02f43504afd.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205450338-79f6080e-cee4-44af-bbc0-037196ca3894.png)

Reply:
@Ugboaja-Uchechi Hello, thanks, for your review, can I use one image for my background?

Reply:
done